vars/kubernetesDeploy.groovy

```diff
@@ -1,7 +1,7 @@
 import groovy.transform.Field
 
 @Field String STEP_NAME = getClass().getName()
-@Field String METADATA_FILE = 'metadata/kubernetesdeploy.yaml'
+@Field String METADATA_FILE = 'metadata/kubernetesDeploy.yaml'
```

Comment:
If we want to rename the metadata files (I'm not sure about that) we could remove this completely and handle it in the `piperExecuteBin` as the `STEP_NAME` in passed into it.

Reply:
yes, we should do so but I guess this would be step 2 since `piperExecuteBin` would need to be adapted ...